server/routes/index.ts

```diff
@@ -53,6 +53,10 @@ export default function routes(router: Router, services: Services): Router {
     }
   })
 
+  get('/service-provider/referrals/:id/action-plan/form/sessions', (req, res) =>
```

Comment:
We've changed this to be action plan ID rather than referral id - did you forget to stash pop?